src/models/horizontal-scaling-summary/test-data.ts

```diff
@@ -30,6 +34,8 @@ export const testData: TestDependencyDataType<HorizontalScalingSummaryModel> = [
     minReplicas: 3,
     currentCPUUtilizationPercentage: 47,
     targetCPUUtilizationPercentage: undefined,
+    currentMemoryUtilizationPercentage: 17,
+    targetMemoryUtilizationPercentage: 33,
```

Comment:
if model is updated, tests needs to be updated
```suggestion
    minReplicas: undefined,
```